Jenkinsfile

```diff
@@ -127,7 +127,8 @@ pipeline {
         agent{  label "tunatest" }
         steps {
           script {
-            utils.solverAnalyticsTest()
+            //utils.solverAnalyticsTest()
+            echo "Need to update paths in SolverAnalytics repo"
```

Comment:
Those become
tuna.miopen.db.tables
tuna.miopen.utils.config_type
tuna.dbBase.sql_alchemy
tuna.utils.db_utility

Once those are updated we can re-enable it on this branch.
We might should add a version here in the Tuna CI, because this update is going to break past builds.

Reply:
Updated solver analytics! You may let the solveranalytics tests run now.
Agreed on adding a version number in the CI